identity_iota_core/src/document/iota_document.rs

```diff
@@ -819,4 +863,149 @@ mod tests {
       format!("{{\"doc\":{},\"meta\":{}}}", document.document, document.metadata)
     );
   }
+
+  #[test]
+  fn deserializing_id_from_other_method_fails() {
```

Comment:
Prefix the newly added tests with `test_`.

Reply:
Why? I never understood why we do that. From a code point of view it is annotated with "#[test]" so I know it is a test and `cargo test` doesn't exercise anything else (apart from doc tests which are also highlighted as such in the terminal).

Reply:
Fair enough, I'm mainly interested in consistency / following conventions here, but I guess it's fine either way. Disregard my comment :)